specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/examples/VmssVmNetworkInterfaceList.json

```diff
@@ -1,6 +1,6 @@
 {
   "parameters" : {
-    "api-version" : "2017-03-30",
+    "api-version": "2017-03-30",
```

Comment:
Shouldn't all versions be `2018-04-01` now? Please update all the examples here accordingly

Reply:
@dsgouda  as I know we shouldn't update VMSS

Reply:
@dsgouda  but they are the part already from previous months... The only change we made is identity change... 
+ @lmazuel  could you please review what's the expected VMSS examples behavior here? As I remember you told me don't touch VMSS :)

Reply:
@MikhailTryakhov @dsgouda The way VMSS in Network is designed, they have to copy the file **as-is** from one month to another. This implies copy the examples with it.
If we want to avoid that copy eventually, it requires a more deeper discussion on how we structure the Swagger for Network and the peculiar VMSS. I don't think that's the point of this PR, and shouldn't be address here.

Reply:
@lmazuel so we should leave api version as is without api-version update? Agree ;)

Reply:
@lmazuel agree but examples that do not match to the current version is a bit moot when being tested. Definitely needs a rethink.
@MikhailTryakhov please mark this as something we need to discuss in upcoming meetings.
Won't block on this

Reply:
@dsgouda  done. please approve/merge this PR?